scripts/apps/vocabularies/controllers/VocabularyEditController.js

```diff
@@ -28,6 +29,11 @@ export function VocabularyEditController($scope, gettext, notify, api, vocabular
             if (angular.isDefined(response.data._issues['validator exception'])) {
                 notify.error(gettext('Error: ' +
                                      response.data._issues['validator exception']));
+            } else if (response.data._issues.error.required_field) {
+                let params = response.data._issues.params;
+
+                notify.error($interpolate(gettext(
+                    'Required {{field}} in item {{item}}'))({field: params.field, item: params.item}));
```

Comment:
```
`How about ${native_es6_interpolation} ?`
```

Reply:
not sure if that works with gettext string extraction

Reply:
Didn't see that. I think it might not.